lib/jose-jwe-webcryptographer.js

```diff
@@ -251,10 +271,10 @@ JoseJWE.WebCryptographer.prototype.decrypt = function(cek_promise, aad, iv, ciph
       return compare(config, mac_key_promise, new Uint8Array(mac), tag).then(function() {
         var dec = {
           name: config.id.name,
-          iv: iv,
+          iv: iv
         };
         return crypto.subtle.decrypt(dec, enc_key, cipher_text);
-      }).catch(function (err) {
+      }).catch(function() {
```

Comment:
imho it's nicer to keep function(err) even if err is unused so the next person reading/debugging the code knows about it. Rename to _err to indicate that it's purposefully unused?